crates/re_types_builder/src/codegen/cpp/mod.rs

```diff
@@ -1226,6 +1232,33 @@ fn component_to_data_cell_method(
     }
 }
 
+fn archetype_indicator(
+    type_ident: &Ident,
+    hpp_includes: &mut Includes,
+    cpp_includes: &mut Includes,
+) -> Method {
+    hpp_includes.insert_rerun("data_cell.hpp");
+    hpp_includes.insert_rerun("arrow.hpp");
+    hpp_includes.insert_rerun("component_batch.hpp");
+    cpp_includes.insert_rerun("indicator_component.hpp");
+
+    Method {
+        docs: "Creates an `AnonymousComponentBatch` out of the associated indicator component. \
```

Comment:
This isn't anonymous if it has an indicator, is it?

Reply:
Not sure what you mean. `Anonymous` is what the cpp SDK calls an erased `ComponentBatch`.

Are you saying you'd like this to return a non-erased `ComponentBatch`, or just that the `Anonymous` prefix sounds weird?